### PR TITLE
chore: Electron 15 blog post

### PR DESCRIPTION
**Pull request by VerteDinde**
First blog post on the new website repo!

Attn @sofianguy

Please add a comment for items you think we should highlight in the blog post; I'll also give your contributor access to my fork if you'd like to push to this PR directly.

⚠️ do not merge until the following are completed ⚠️

- [x]  update node, v8 and chromium versions from final chrome roll under Stack Changes section
- [x]  add a couple new features in tag line sentence
- [x]  edit link for M94 blog post
- [x]  add a few bullets for Highlight Features section
- [x]  add any missing items in Breaking Changes section
- [x]  add items for API Changes section
- [x]  add items for Deprecated APIs section

### CODE REVIEW

File: blog/electron-15-0.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+### Highlight Features
+
+* 
+
+See the [15.0.0 release notes](https://github.com/electron/electron/releases/tag/v15.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+### Default Changed: nativeWindowOpen defaults to true
+Prior to Electron 15, window.open was by default shimmed to use BrowserWindowProxy. This meant that window.open('about:blank') did not work to open synchronously scriptable child windows, among other incompatibilities. nativeWindowOpen: true is no longer experimental, and is now the default.
```
Comment (sofianguy) [inline blog/electron-15-0.md:48]:
```suggestion
Prior to Electron 15, `window.open` was by default shimmed to use `BrowserWindowProxy`. This meant that `window.open('about:blank')` did not work to open synchronously scriptable child windows, among other incompatibilities. `nativeWindowOpen: true` is no longer experimental, and is now the default.
```

File: blog/electron-15-0.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+See the documentation for window.open in Electron for more details.
```
Comment (sofianguy) [inline blog/electron-15-0.md:50]:
```suggestion
See the documentation for [window.open](https://www.electronjs.org/docs/api/window-open) in Electron for more details.
```

File: blog/electron-15-0.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    * [V8 9.4 blog post](https://v8.dev/blog/v8-release-94)
+
+### Highlight Features
+
+* 
+
+See the [15.0.0 release notes](https://github.com/electron/electron/releases/tag/v15.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
```
Comment (sofianguy) [inline blog/electron-15-0.md:46]:
```suggestion

Below are breaking changes introduced in Electron 15. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/breaking-changes) page.
```

File: blog/electron-15-0.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+### Default Changed: nativeWindowOpen defaults to true
+Prior to Electron 15, window.open was by default shimmed to use BrowserWindowProxy. This meant that window.open('about:blank') did not work to open synchronously scriptable child windows, among other incompatibilities. nativeWindowOpen: true is no longer experimental, and is now the default.
+
+See the documentation for window.open in Electron for more details.
+
+More information about future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/breaking-changes) page.
```
Comment (sofianguy) [inline blog/electron-15-0.md:52]:
```suggestion

```
Moving this line up so it doesn't appear to be part of the nativeWindowOpen header

File: blog/electron-15-0.md (hunk 2 above)
Comment (erickzhao) [reply blog/electron-15-0.md:50]: @molant wonder if we should put the beta docs up in these links :)

Comment (molant) [reply blog/electron-15-0.md:50]: All the links in the site point to `/docs/latest` so I wouldn't call them beta anymore 😅

I'd prefer if we could use them 😊

**Review by VerteDinde — COMMENTED** [review]
Adding a note here to hold before we potentially merge - I'm not sure that nativeWindowOpen was ever set to `true` by default in Electron 15. I was only able to find a PR reverting the original change to set it to default in E14 [[#29869](https://github.com/electron/electron/pull/29869)], and an open API PR to remove the option targeting 16 [[#29405](https://github.com/electron/electron/pull/29405)]. We have it in our Planned Breaking Changes doc for E15, but I think that was set before we changed the release timeline, just want to make sure I update that doc and this post if it's outdated.

File: blog/electron-15-0.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+See the documentation for [window.open](https://www.electronjs.org/docs/api/window-open) in Electron for more details.
+
+
+## API Changes
+
+* Added 'frame-created' event to `WebContents` which emits when a frame is created in the page. [#30801](https://github.com/electron/electron/pull/30801) 
+* Added `safeStorage` string encryption API. [#30430](https://github.com/electron/electron/pull/30430) 
+* Added `signal` option to `dialog.showMessageBox`. [#26102](https://github.com/electron/electron/pull/26102) 
+* Added an [Electron Fuse](https://www.electronjs.org/docs/tutorial/fuses) for enforcing code signatures on the `app.asar` file your application loads.  Requires the latest `asar` module. [#30900](https://github.com/electron/electron/pull/30900)
```
Comment (erickzhao) [inline blog/electron-15-0.md:63]: Might be clear to mention which semver of asar is required here.

File: blog/electron-15-0.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+Prior to Electron 15, `window.open` was by default shimmed to use `BrowserWindowProxy`. This meant that `window.open('about:blank')` did not work to open synchronously scriptable child windows, among other incompatibilities. `nativeWindowOpen: true` is no longer experimental, and is now the default.
+
+See the documentation for [window.open](https://www.electronjs.org/docs/api/window-open) in Electron for more details.
+
+
+## API Changes
+
+* Added 'frame-created' event to `WebContents` which emits when a frame is created in the page. [#30801](https://github.com/electron/electron/pull/30801) 
+* Added `safeStorage` string encryption API. [#30430](https://github.com/electron/electron/pull/30430)
```
Comment (erickzhao) [inline blog/electron-15-0.md:61]: I'd add this to the Highlight Features because it exposes a whole new module to the main process.

Comment (VerteDinde) [reply blog/electron-15-0.md:61]: That's a good point, will do - and it gives George a highlighted feature shout out! 🙂

File: blog/electron-15-0.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+Starting with Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
+
+Additionally, Electron will be changing supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://electronjs.org/docs/tutorial/electron-versioning)for more detailed information about versioning in Electron.
```
Comment (molant) [inline blog/electron-15-0.md:28]:
```suggestion
Additionally, Electron will be changing supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://www.electronjs.org/docs/latest/tutorial/electron-versioning)for more detailed information about versioning in Electron.
```

File: blog/electron-15-0.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+* `nativeWindowOpen: true` is no longer experimental, and is now the default.
+* Added 'frame-created' event to `WebContents` which emits when a frame is created in the page. [#30801](https://github.com/electron/electron/pull/30801) 
+* Added resize `edge` info to `BrowserWindow`'s `will-resize` event. [#29199](https://github.com/electron/electron/pull/29199)
+
+See the [15.0.0 release notes](https://github.com/electron/electron/releases/tag/v15.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+Below are breaking changes introduced in Electron 15. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/breaking-changes) page.
```
Comment (molant) [inline blog/electron-15-0.md:49]:
```suggestion
Below are breaking changes introduced in Electron 15. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/latest/breaking-changes) page.
```

File: blog/electron-15-0.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+### Default Changed: nativeWindowOpen defaults to true
+
+Prior to Electron 15, `window.open` was by default shimmed to use `BrowserWindowProxy`. This meant that `window.open('about:blank')` did not work to open synchronously scriptable child windows, among other incompatibilities. `nativeWindowOpen: true` is no longer experimental, and is now the default.
+
+See the documentation for [window.open](https://www.electronjs.org/docs/api/window-open) in Electron for more details.
```
Comment (molant) [inline blog/electron-15-0.md:55]:
```suggestion
See the documentation for [window.open](https://www.electronjs.org/docs/latest/api/window-open) in Electron for more details.
```

File: blog/electron-15-0.md (hunk 5 above)
Comment (molant) [reply blog/electron-15-0.md:63]:
```suggestion
* Added an [Electron Fuse](https://www.electronjs.org/docs/latest/tutorial/fuses) for enforcing code signatures on the `app.asar` file your application loads.  Requires the latest `asar` module. [#30900](https://github.com/electron/electron/pull/30900) 
```

File: blog/electron-15-0.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+|	15.x.y |	16.x.y |	17.x.y |	18.x.y |	19.x.y |
+|	14.x.y |	15.x.y |	16.x.y |	17.x.y |	18.x.y |
+|	13.x.y |	14.x.y |	15.x.y |	16.x.y |	17.x.y |
+|	12.x.y |	13.x.y |	14.x.y |	15.x.y |	-- |
+
+## What's Next
+
+In the short term, you can expect the team to continue to focus on keeping up with the development of the major components that make up Electron, including Chromium, Node, and V8. Although we are careful not to make promises about release dates, our plan is release new major versions of Electron with new versions of those components approximately quarterly.
+
+You can find [Electron's public timeline here](https://electronjs.org/docs/tutorial/electron-timelines).
```
Comment (molant) [inline blog/electron-15-0.md:92]:
```suggestion
You can find [Electron's public timeline here](https://www.electronjs.org/docs/latest/tutorial/electron-timelines).
```

File: blog/electron-15-0.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+slug: electron-15-0
+---
+
+Electron 15.0.0 has been released! It includes upgrades to Chromium `94`, V8 `9.4`, and Node.js `16.5.0`. We've added API updates to window.open, bug fixes, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
```
Comment (molant) [inline blog/electron-15-0.md:20]:
```suggestion
The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
```

File: blog/electron-15-0.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-15-0
+---
+
+Electron 15.0.0 has been released! It includes upgrades to Chromium `94`, V8 `9.4`, and Node.js `16.5.0`. We've added API updates to window.open, bug fixes, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
```
Comment (molant) [inline blog/electron-15-0.md:18]:
```suggestion
The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
```

**Review by molant — COMMENTED** [review]
As mentioned in one of the comments we should be using the `/docs/latest` route as the other is being deprecated.
I also suggested adding `www.` to the URLs as that's where users are redirected and thus avoiding a hop.

File: blog/electron-15-0.md (hunk 5 above)
Comment (VerteDinde) [reply blog/electron-15-0.md]: @erickzhao Added asar semver version v3.1.0 or higher 👍

**Review by molant — APPROVED** [review]

**Pull request merged**